modules/distribution/resources/stat/migration-1.9.0_to_1.10.0/migration.sql

```diff
@@ -0,0 +1,250 @@
+
+  
+create temporary table APIThrottleSummaryData using CarbonJDBC options (dataSource "WSO2AM_STATS_DB", tableName "API_THROTTLED_OUT_SUMMARY");
+CREATE TEMPORARY TABLE THROTTLED_OUT_FINAL_SUMMARY USING CarbonAnalytics OPTIONS (tableName "API_THROTTLED_OUT_SUMMARY",
+  schema "api string -i,
+  api_version string -i,
+  context string -i,
+  apiPublisher string -i,
+  applicationName string -i,
+  tenantDomain string -i,
+  year int -i,
+  month int -i,
+  day int -i,
+  week int -i,
+  time string -i,
+  success_request_count int -i,
+  throttleout_count int -i,
+  max_request_time long -i,
+  api_apiPublisher_applicationName_facet facet -i,
+  applicationName_facet facet -i",
+  primaryKeys "api,api_version,context,apiPublisher,applicationName,tenantDomain,year,month,day"
+);
+
+insert into table THROTTLED_OUT_FINAL_SUMMARY select 
+api,api_version,context,apiPublisher,applicationName,tenantDomain,year,month,
+day,week,time,COALESCE(success_request_count,0),COALESCE(throttleout_count,0),
+COALESCE(timestamp(time),0),concat( api,concat(",",concat(apiPublisher,concat(",",applicationName)) )),
+applicationName from APIThrottleSummaryData; 
+
+
+
+
+                            
+
+
+  
+
+  CREATE TEMPORARY TABLE API_REQUEST_SUMMARY_FINAL USING CarbonAnalytics OPTIONS (tableName "API_REQUEST_SUMMARY",
+  schema "api string -i,
+  api_version string -i,
+  version string -i,
+  apiPublisher string -i,
+  consumerKey string -i,
+  userId string -i,
+  context string -i,
+  max_request_time long -i,
+  total_request_count int -i,
+  hostName string -i,
+  year int -i,
+  month int -i,
+  day int -i,
+  time string -i,
+  key_api_facet facet -i,
+  key_userId_facet facet -i,
+  api_version_userId_facet facet -i,
+  api_version_userId_apiPublisher_facet facet -i,
+  api_version_userId_context_facet facet -i",
+  primaryKeys "api,api_version,version,apiPublisher,consumerKey,userId,context,hostName,year,month,day"
+  );
+
+  create temporary table APIRequestSummaryData using CarbonJDBC options (dataSource "WSO2AM_STATS_DB", tableName "API_REQUEST_SUMMARY");
+
+  insert into table API_REQUEST_SUMMARY_FINAL select api, api_version,version, apiPublisher,
+  consumerKey,
+  userId,
+  context,
+  max_request_time,
+  total_request_count,
+  hostName,
+  year,
+  month,
+  day,
+  time,
+  concat(concat(consumerKey,","),api),
+  concat(concat(consumerKey,","),userId),
+  concat(concat(api,","),concat(concat(version,","),userId)),
+  concat(concat(api,","),concat(concat(version,","),concat(concat(userId,","),apiPublisher))),
+  concat(concat(api,","),concat(concat(version,","),concat(concat(userId,","),context)))
+  from APIRequestSummaryData;
+
+  
```

Comment:
fixed in commit: https://github.com/wso2/product-apim/commit/82d72302e4ea89d194dd3cc54878bfaedb83e926